kasenvr/project-athena
# Adds interim decentralizedGoTo as a default script, adds beta channel as well.

Currently the beta channel has no toggle to activate it but at least the system is in place to do so.

To test:

Use the goto ("Explore" app) and confirm it works and loads the default JSON (which is from metaverse.projectathena.io).

Confirm all of your other default apps load and work as expected.
